lib/process-manager.js

```diff
@@ -60,6 +63,33 @@ class ProcessManager {
         // Base Implementation
     }
 
+    /**
+     * General implementation of figuring out if the Ghost blog has started successfully.
+     *
+     * @returns {Promise<any>}
+     */
+    ensureStarted(options) {
```

Comment:
we should require the port polling utility inside this function, so `ghost run` doesn't have to require it.

Reply:
👍